qxf2_scheduler/qxf2_scheduler.py

```diff
@@ -46,14 +48,52 @@ def is_qxf2_holiday(date):
 
     return result_flag
 
+
 def is_weekend(date):
     "Is this a weekend?"
     date = gcal.process_date_string(date)
     day = date.weekday()
-    print('~~~~',day)
+    
     return True if day==5 or day==6 else False
 
 
+def convert_combined_string_into_isoformat(create_event_timings_and_date):
+    "Converting the string into iso format"
+    converted_create_event_date_and_time = datetime.datetime.strptime(create_event_timings_and_date,"%m/%d/%Y%H:%M").isoformat()
```

Comment:
The format used here should be stored as a global constant. That way, when we make a change, we will change things only in one place.

Reply:
Done